src/engine/tests/Test_ReluConstraint.h

```diff
@@ -58,11 +60,13 @@ class ReluConstraintTestSuite : public CxxTest::TestSuite
     void setUp()
     {
         TS_ASSERT( mock = new MockForReluConstraint );
+        TS_ASSERT( bm = new BoundManager( ctx ) );
```

Comment:
This can be removed if the tests use MockTableau instead of a BM. Same with line 69.